index.js

```diff
@@ -76,11 +29,7 @@ function fastifyEH (fastify, opts, next) {
       },
       add: function (content) {
         const p = new Promise((resolve) => {
-          if (reply.raw.socket) {
```

Comment:
Help me remember, this is a Fastify 3 way right?

Reply:
I reverted this. Description below